### PR TITLE
catkinized cob_perception_common

### PR DESCRIPTION
**Pull request by mgruhler**
please test this :-)

### CODE REVIEW

File: cob_vision_utils/manifest.xml [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
 
-     cob_vision_utils
-
-  </description>
-  <author>Jan Fischer</author>
-  <license>LGPL</license>
-  <review status="unreviewed" notes=""/>
-  <url>http://ros.org/wiki/cob_vision_utils</url>
 
   <depend package="roscpp"/>
```
Comment (ipa-goa) [inline cob_vision_utils/manifest.xml]: file seems to be corrupt, delete the whole file?

**Comment by floweisshardt** [comment]
why do we need a groovy_catkin branch? catkin development should only be done for hydro and above, there's no catkin support for all other repositories for catkin under groovy

**Comment by mgruhler** [comment]
Because we need that in another project, under groovy.
Also, this is a first step and can (almost directly) be used as the hydro_dev branch (except the PCL dependency).

---

Dipl.-Ing. Matthias GRUHLER
Wissenschaftlicher Mitarbeiter
Research Assistant

Fraunhofer-Institut für Produktionstechnik und Automatisierung IPA

Abteilung Roboter- und Assistenzsysteme
Department Robot and Assistive Systems

Gruppe Industrielle und gewerbliche Servicerobotik
Group Industrial and Commercial Service Robotics

Nobelstraße 12 │ 70569 Stuttgart
Phone +49 711 970-1315
Fax +49 711 970-1008

Matthias.Gruhler@ipa.fraunhofer.demailto:Matthias.Gruhler@ipa.fraunhofer.de
www.ipa.fraunhofer.dehttp://www.ipa.fraunhofer.de
www.care-o-bot.dehttp://www.care-o-bot.de

Von: Florian Weißhardt [mailto:notifications@github.com]
Gesendet: Freitag, 31. Januar 2014 15:30
An: ipa320/cob_perception_common
Cc: Gruhler, Matthias
Betreff: Re: [cob_perception_common] catkinized cob_perception_common (#14)

why do we need a groovy_catkin branch? catkin development should only be done for hydro and above, there's no catkin support for all other repositories for catkin under groovy

**Comment by floweisshardt** [comment]
ok.

**Pull request merged**